aeron-system-tests/src/test/java/io/aeron/cluster/MultiModuleSharedDriverTest.java

```diff
@@ -126,6 +127,8 @@ public void shouldSupportTwoSingleNodeClusters()
                 final EgressListener egressListener = (clusterSessionId, timestamp, buffer, offset, length, header) ->
                     egress.set(buffer.getStringWithoutLengthAscii(offset, length));
 
+                Tests.sleep(1_000);
```

Comment:
Why is this needed?

Reply:
It's not, that was me trying to debug a failure.  Removed.